core/src/AzureMcp.Cli/Program.cs

```diff
@@ -67,24 +67,26 @@ private static IAreaSetup[] RegisterAreas()
             new AzureMcp.AppConfig.AppConfigSetup(),
             new AzureMcp.Authorization.AuthorizationSetup(),
             new AzureMcp.AzureIsv.AzureIsvSetup(),
-            new AzureMcp.Cosmos.CosmosSetup(),
             new AzureMcp.Foundry.FoundrySetup(),
             new AzureMcp.Grafana.GrafanaSetup(),
             new AzureMcp.KeyVault.KeyVaultSetup(),
             new AzureMcp.Kusto.KustoSetup(),
             new AzureMcp.Marketplace.MarketplaceSetup(),
-            new AzureMcp.Monitor.MonitorSetup(),
-            new AzureMcp.Postgres.PostgresSetup(),
             new AzureMcp.Redis.RedisSetup(),
-            new AzureMcp.Search.SearchSetup(),
             new AzureMcp.ServiceBus.ServiceBusSetup(),
             new AzureMcp.Sql.SqlSetup(),
             new AzureMcp.Storage.StorageSetup(),
             new AzureMcp.Workbooks.WorkbooksSetup(),
-            new AzureMcp.BicepSchema.BicepSchemaSetup(),
-            new AzureMcp.VirtualDesktop.VirtualDesktopSetup(),
             new AzureMcp.AzureTerraformBestPractices.AzureTerraformBestPracticesSetup(),
             new AzureMcp.LoadTesting.LoadTestingSetup(),
+#if !BUILD_NATIVE
```

Comment:
nit: it would be nice if there were a source comment referencing the GitHub issue number(s) tracking the problems